generate-roles.go

```diff
@@ -0,0 +1,39 @@
+package main
```

Comment:
It'd be nice to not enforce go as a dependency for building libuast, but I don't think to have a way to generate the roles without it for now, so it's ok.